utils/polygonRequest.py

```diff
@@ -16,6 +15,10 @@ class PolygonRequest:
     polygon_format = '&format=jsonv2&polygon_geojson=1'
     API_url = 'https://nominatim.openstreetmap.org/search?q='
```

Comment:
concat *API_url* & *polygon_format* to same string 
like this :
API_URL = 'https://nominatim.openstreetmap.org/search?format=jsonv2&polygon_geojson=1&q='

Reply:
and thus remove *self.polygon_format* line 37, since all the url arguments would be at the beginning of the string